.travis.yml

```diff
@@ -1,21 +1,20 @@
 language: python
-
+sudo: required
```

Comment:
https://travis-ci.org/zalando/ghe-backup/builds/203975546/#L290 causes ```sudo required``` IMO. 
see also commit message of https://github.com/zalando/ghe-backup/pull/43/commits/f4c61e1d0910ce6a51caccf1296f118ea980a4df.